src/Surfnet/Stepup/Configuration/Value/AllowedSecondFactorList.php

```diff
@@ -19,11 +19,12 @@
 namespace Surfnet\Stepup\Configuration\Value;
 
 use ArrayIterator;
+use Broadway\Serializer\SerializableInterface;
 use IteratorAggregate;
 use JsonSerializable;
 use Surfnet\StepupBundle\Value\SecondFactorType;
 
-final class AllowedSecondFactorList implements JsonSerializable, IteratorAggregate
+final class AllowedSecondFactorList implements JsonSerializable, IteratorAggregate, SerializableInterface
```

Comment:
Just a thought: personally, I prefer to have interface names listed by alphabet. Really is something minor, but I've noticed that mental overhead is lower and that code when adding an interface provides cleaner diffs. Plus expectation on what is listed where etc.  Really can't think of a single truly compelling argument, mostly preference and some minor advantages.